src/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -567,6 +567,10 @@ public static class MsalError
         /// </summary>
         public const string LinuxXdgOpen = "linux_xdg_open_failed";
 
+        /// <summary>
+        /// The selected webview is not available on this platform. You can switch to a different webview using .WithUseEmbeddedWebview(bool). See https://aka.ms/msal-net-os-browser for details
+        /// </summary>
+        public const string WebviewUnavailable = "no_system_webview";
```

Comment:
>WebviewUnavailable [](start = 28, length = 18)

when we brought embedded webview to MSAL there were lots of discussions w/android team about naming. they were particular about using "system browser" and not "system webview"...not sure if it matters here, but wanted to bring it up.

Reply:
Yes, but our public API has the term WithEmbeddedWebView so I prefer to use this consistently.